pkg/api/http/scheduler.go

```diff
@@ -71,20 +72,20 @@ func (a *api) onCreateScheduleHandler() http.HandlerFunc {
 		UniversalHTTPHandlerOpts[*runtimev1pb.ScheduleJobRequest, *emptypb.Empty]{
 			InModifier: func(r *http.Request, in *runtimev1pb.ScheduleJobRequest) (*runtimev1pb.ScheduleJobRequest, error) {
 				// Users should set the name in the url, and not in the url and body
-				if (chi.URLParam(r, nameParam) == "") || (in.GetJob().GetName() != "") {
+				name := strings.TrimSpace(chi.URLParam(r, nameParam))
+				if len(name) == 0 {
+					apierrors.Empty("Job", map[string]string{"appId": a.universal.AppID()}, apierrors.ConstructReason(apierrors.CodePrefixScheduler, apierrors.PostFixEmpty))
+				}
+				if in.GetJob().GetName() != "" {
 					return nil, apierrors.SchedulerURLName(map[string]string{"appId": a.universal.AppID()})
```

Comment:
ik you didn't do this, this was probably me, but can you change it? I thought we were standardizing on `appID`
```suggestion
					return nil, apierrors.SchedulerURLName(map[string]string{"appID": a.universal.AppID()})
```